docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -206,7 +210,7 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 
        .. code-block:: bash
 
-          scripts/core/upgrade-listener
+          vctl install examples/ListenerAgent
```

Comment:
Note this does not use the config file at all so you might add --agent-config examples/ListenerAgent/config